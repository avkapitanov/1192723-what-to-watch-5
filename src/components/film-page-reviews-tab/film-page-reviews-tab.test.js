import React from "react";
import renderer from "react-test-renderer";
import FilmPageReviewsTab from "./film-page-reviews-tab";

const reviews = [
  {
    id: 1,
    user: {
      id: 16,
      name: `Mollie`
    },
    rating: 3.3,
    comment: `I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.`,
    date: `2020-10-30T11:53:40.154Z`
  },
  {
    id: 2,
    user: {
      id: 15,
      name: `Kendall`
    },
    rating: 9.4,
    comment: `I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.`,
    date: `2020-10-13T11:53:40.154Z`
  },
  {
    id: 3,
    user: {
      id: 13,
      name: `Zak`
    },
    rating: 3.3,
    comment: `This movie is perfect in all its categories: credits, sound track, production, casting, writing, photography, editing, acting, and direction. I was amazed with the freedom of the use of the camera. This movie will change the way movies are made. Slow-mo, stills, black and white, and color were all used to brilliant effect.`,
    date: `2020-10-06T11:53:40.154Z`
  }
];

it(`FilmPageReviewsTab component render correctly`, () => {
  const tree = renderer.create(
      <FilmPageReviewsTab
        reviews={reviews}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
