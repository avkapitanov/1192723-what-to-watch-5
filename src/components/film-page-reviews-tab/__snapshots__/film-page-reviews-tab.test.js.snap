// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmPageReviewsTab component render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Mollie
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-30"
          >
            October 30, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        3.3
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Kendall
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-13"
          >
            October 13, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        9.4
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          This movie is perfect in all its categories: credits, sound track, production, casting, writing, photography, editing, acting, and direction. I was amazed with the freedom of the use of the camera. This movie will change the way movies are made. Slow-mo, stills, black and white, and color were all used to brilliant effect.
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Zak
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-06"
          >
            October 06, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        3.3
      </div>
    </div>
  </div>
</div>
`;
